src/OrchardCore.Modules/OrchardCore.Taxonomies/Controllers/TagController.cs

```diff
@@ -18,29 +18,32 @@
 namespace OrchardCore.Taxonomies.Controllers
 {
     [Admin]
-    public class TagController : Controller, IUpdateModel
+    public class TagController : Controller
     {
         private readonly IContentManager _contentManager;
         private readonly IAuthorizationService _authorizationService;
         private readonly IContentDefinitionManager _contentDefinitionManager;
         private readonly IEnumerable<IContentHandler> _contentHandlers;
         private readonly ISession _session;
         private readonly ILogger _logger;
+        private readonly IUpdateModelAccessor _updateModelAccessor;
 
         public TagController(
             IContentManager contentManager,
             IAuthorizationService authorizationService,
             IContentDefinitionManager contentDefinitionManager,
             IEnumerable<IContentHandler> contentHandlers,
             ISession session,
-            ILogger<TagController> logger)
+            ILogger<TagController> logger,
+            IUpdateModelAccessor updateModelAccessor)
```

Comment:
We usually inject the logger at the end but anyway `_updateModelAccessor` is not used afterwards

So remove the injection of `IUpdateModelAccessor` and its related field